oat-sa/extension-tao-task-queue
# fix: return types should be int.

## What's Changed
- Fixed return type of oat\taoTaskQueue\model\QueueBroker\RdsQueueBroker::count().
- Fixed return type of oat\taoTaskQueue\model\QueueBroker\SqsQueueBroker::count().

## TODO 


## How to test
- Run worker process
- Watch logs:
`tail -f /var/www/html/tao.log`
- No errors should be visible in logs:
`Return type of oat\taoTaskQueue\model\QueueBroker\RdsQueueBroker::count() should either be compatible with Countable::count(): int, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice`
`Return type of oat\taoTaskQueue\model\QueueBroker\SqsQueueBroker::count() should either be compatible with Countable::count(): int, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice`
